lib/trivial.nix

```diff
@@ -389,4 +389,17 @@ rec {
       assert (base >= 2);
       assert (i >= 0);
       lib.reverseList (go i);
+
+  /* If `cond' is true, return the value `because', otherwise `false'. Useful
+     for setting `meta.broken' with a descriptive message (e.g.
+     `meta.broken = optionalBroken disable "${version} is not supported";').
+
+     Example:
+       optionalBroken true "message"
+       => "message"
+       optionalBroken false "message"
+       => false
+  */
+  optionalBroken = cond: because:
+    if cond then because else false;
```

Comment:
lets use optionalString instead

Reply:
Using `optionalString` is possible, but I tried to avoid a second `not broken` value (`false` and `""`) and `optionalBroken` was my workaround.

But I admit, using `optionalString` would remove the need for a new `lib` function and perform the same task.